packages/pwa-buildpack/src/WebpackTools/MagentoResolver.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
         return {
             modules: [options.paths.root, 'node_modules'],
             mainFiles: ['index'],
-            extensions: ['.js', '.json']
+            extensions: ['.mjs', '.js', '.json', '.graphql']
```

Comment:
We don't, but `node_modules` does, and with Webpack 4 tree shaking set up correctly, we will directly access those modules rather than using their Babelified `dist` versions.